### DISCUSSION TITLE
scanner functionallity

### DISCUSSION THREAD

**Discussion by justsomeguyontheinternet** [minio/minio#13828] [Q&A] [answered]
hi.

what does the scanner do? what benefit does it provide in my use case? can it be disabled enitrely?

i have a docker single instance of minio on a home server with only a single disk. minio is used for hosting backup shards only with ~5 clients backuping up to minio daily. sometime in the last year i think the scanner behavior was changed, and now minio is always using the disk for background scanning, it basically never idles.

**Comment by harshavardhana** [comment]
> i have a docker single instance of minio on a home server with only a single disk. minio is used for hosting backup shards only with ~5 clients backuping up to minio daily. sometime in the last year i think the scanner behavior was changed, and now minio is always using the disk for background scanning, it basically never idles.

Are you saying you are okay to not have healing and verification on your objects? do you not care about your objects?

**Reply by justsomeguyontheinternet** [reply]
haha of course i care! just not that much....

in my case the minio server is hosting backups. verification is handled by the backup application (duplicati). the backups go to multiple servers in multiple locations.

as i understand with my usage, single disk, no erasure coding, single instance, all the scanner is doing is getting disk usage since healing and verification won't work. the scanner doesn't seem worth it in my case.

i think i will delay the runs significantly, but wanted to make sure i understood the impact.

thanks!

**Comment by harshavardhana** [comment, marked as answer]
> what does the scanner do? what benefit does it provide in my use case? can it be disabled enitrely?

Read what it does here https://github.com/minio/minio/tree/master/docs/config#usage-scanner - you can't turn it off. Sorry! you can delay the runs.

**Reply by klauspost** [reply]
You can tweak scanner resource use by using `mc admin config set ALIAS scanner speed=default.`

Possible values are: fastest, fast, default, slow, slowest

**Reply by adababys** [reply]
> You can tweak scanner resource use by using `mc admin config set ALIAS scanner speed=default.`
> 
> Possible values are: fastest, fast, default, slow, slowest

"Hello, the command "mc admin config set ALIAS scanner speed=default " does not work in my environment. I am using MinIO version 202208xx and mc version 202308xx."

**Reply by klauspost** [reply]
Then upgrade.
